### DISCUSSION TITLE
Is MSSQL fully supported?

### DISCUSSION THREAD

**Discussion by acelaya** [mikro-orm/mikro-orm#5605] [General]
I see there's what seems to be an up to date and official [MSSQL driver](https://www.npmjs.com/package/@mikro-orm/mssql), however, [the documentation](https://mikro-orm.io/docs/guide#mikroorm-monorepo) does not mention it in the "Full list of currently available drivers"

![image](https://github.com/mikro-orm/mikro-orm/assets/2719332/157f3bd2-b75e-48d2-9262-04abce54e711)

So I just wanted to double check. Is it supported and intended to continue being supported in future, and this is just an overlook in the docs?

**Comment by acelaya** [comment]
I just realized mssql is mentioned in this other page https://mikro-orm.io/docs/quick-start, so I assume the other one is just an overlook.

![image](https://github.com/mikro-orm/mikro-orm/assets/2719332/35fdc10c-0b00-4f53-a15f-53d95bda6193)

**Comment by B4nan** [comment]
That list is just outdated, SQL server is supported (although some features are limited due to how it works) and the support won't go anywhere :] I'll update the list in the getting started guide, it was created before the new drivers came in.

**Reply by acelaya** [reply]
Thanks for the confirmation 🙂
